grails-app/conf/logback.xml

```diff
@@ -1,5 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    <include resource="org/springframework/boot/logging/logback/console-appender.xml"/>
```

Comment:
This is what gets imported here: https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/console-appender.xml

It saves us from having to define our own console appender since we're just using the default config anyways. Then we can override the format with the `logging.pattern.console:` property which maps to CONSOLE_LOG_PATTERN which is defined in https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/defaults.xml